lib/disasters.js

```diff
@@ -145,7 +168,82 @@ function getDisaster(disasterId, callback) {
   });
 }
 
+async.auto({
```

Comment:
I'm guessing this part was added for testing the disasters notification alone. If so, it should be removed before it's pushed to dev.